### PR TITLE
Unhide tctl alert create

### LINKED ISSUE #6442: Maintenance Mode/Announcement Banner
### What
Provide a free form text  field  to display text to all users logged into web UI

### How
Provide an option in teleport.yaml under the proxy section
```
display_banner:
      from_time:
      to_time:
      banner_text:
```
Alternatively also,provide an option in the web UI to update banner text
### Why

This is needed to inform users in advance about  maintenance  windows and/or to make  announcements regarding  service health/status.

### Workaround

None

### PR DESCRIPTION
**Pull request by zmb3**
Also add optional --ttl and --severity flags.
This allows cluster administrators to set custom alerts (during maintenance periods, for example).

Fixes #6442

### CODE REVIEW

**Review by fspmarshall — COMMENTED** [review]
Alerts use labels to decide when they should be shown and who is allowed to see them.  An alert with no labels isn't shown to anyone.  In the interest of user-friendliness I think we aught to apply the on-login and permit-all labels if `--labels` is not specified (as seen [here](https://github.com/gravitational/teleport/blob/62f7537d6520477831b5a42c962535d12ad62d1c/lib/auth/auth.go#L555-L556)).  Those two labels combined will produce the behavior that most users would expect from this command I think.

File: tool/tctl/common/alert_command.go [hunk 1]
```diff
@@ -130,12 +133,25 @@ func (c *AlertCommand) Create(ctx context.Context, client auth.ClientI) error {
...
-	alert, err := types.NewClusterAlert(uuid.New().String(), c.message)
+	var sev types.AlertSeverity
+	switch c.severity {
+	case "low":
+		sev = types.AlertSeverity_LOW
+	case "medium":
+		sev = types.AlertSeverity_MEDIUM
+	case "high":
+		sev = types.AlertSeverity_HIGH
+	}
```
Comment (fspmarshall) [inline tool/tctl/common/alert_command.go:144]:
```suggestion
	var sev types.AlertSeverity
	switch c.severity {
	case "low", "":
		sev = types.AlertSeverity_LOW
	case "medium":
		sev = types.AlertSeverity_MEDIUM
	case "high":
		sev = types.AlertSeverity_HIGH
	default:
		return trace.BadParameter("unexpected severity %q, try using one of low, medium, or high", c.severity)
	}
```

Comment (fspmarshall) [reply tool/tctl/common/alert_command.go:144]: Nevermind, just noticed that you set it up as an enum.

Comment (zmb3) [reply tool/tctl/common/alert_command.go:144]: Technically the empty string isn't possible due to the EnumVar and default declaration in the flag, but I will switch the default from medium to low.

**Comment by zmb3** [comment]
Thanks for the pointer on labels - should be good to go now.

**Review by fspmarshall — APPROVED** [review]

**Review by hatched — APPROVED** [review]

**Pull request merged**